docs/api/Dispatcher.md

```diff
@@ -193,7 +193,7 @@ Returns: `Boolean` - `false` if dispatcher is busy and further dispatch calls wo
 * **path** `string`
 * **method** `string`
 * **body** `string | Buffer | Uint8Array | stream.Readable | Iterable | AsyncIterable | null` (optional) - Default: `null`
-* **headers** `UndiciHeaders` (optional) - Default: `null`
+* **headers** `UndiciHeaders | string[]` (optional) - Default: `null`.
```

Comment:
Are you sure that is `string[]`? I mean, looks like the headers must be `string[][]` then you can just `new Headers(headers)`

Reply:
Yes it is, or else I could have already used Object.fromEntries :/

Reply:
I'll expand a little (so that without the test you can verify); if we used the spread operator on a 2d array, the results would be very different:

```js
 // 2d array:
 { ...[ ['a', 'b'], ['c', 'd'] ] } // -> { '0': [ 'a', 'b' ], '1': [ 'c', 'd' ] }
 
 // 1d array
 { ... ['a', 'b', 'c', 'd'] } // -> { '0': 'a', '1': 'b', '2': 'c', '3': 'd' }
}
```